.github/workflows/concrete_python_test_macos.yml

```diff
@@ -119,6 +119,31 @@ jobs:
           cp $GITHUB_WORKSPACE/frontends/concrete-python/Makefile .
           KEY_CACHE_DIRECTORY=./KeySetCache PYTEST_MARKERS="not dataflow and not graphviz" make pytest-macos
 
+      - name: Test notebooks
+        run: |
+          set -e
+          export TEST_TMP_DIR=$(mktemp -d)
+          echo "TEST_TMP_DIR=$TEST_TMP_DIR" >> "${GITHUB_ENV}"
+          cd $TEST_TMP_DIR
+
+          python3.10 -m venv .testenv
+          . .testenv/bin/activate
+
+          pip install $GITHUB_WORKSPACE/frontends/concrete-python/dist/*macos*.whl
+          pip install -r $GITHUB_WORKSPACE/frontends/concrete-python/requirements.dev.txt
+
+          # MacOS x86 have conflict between our OpenMP library, and one from torch
+          # we fix it by using a single one (from torch)
+          # see discussion: https://discuss.python.org/t/conflicting-binary-extensions-in-different-packages/25332/8
+
+          find .testenv/lib/python3.10/site-packages -not \( -path .testenv/lib/python3.10/site-packages/concrete -prune \) -name 'lib*omp5.dylib' -or -name 'lib*omp.dylib' | xargs -n 1  ln -f -s $(pwd)/.testenv/lib/python3.10/site-packages/concrete/.dylibs/libomp.dylib
+
+          cp -R $GITHUB_WORKSPACE/frontends/concrete-python/examples ./examples
+          cp -R $GITHUB_WORKSPACE/frontends/concrete-python/tests ./tests
+
+          cp $GITHUB_WORKSPACE/frontends/concrete-python/Makefile .
+          PYTEST_MARKERS="not dataflow and not graphviz" make test-notebooks
```

Comment:
for macOS as well, per @umut-sahin

Reply:
key cache is not used here? If yes you must specify it else I think it may/will use something which not being cleanup. btw I think we could export PYTEST_MARKERS and KEY_CACHE_DIRECTORY in the GITHUB_ENV like TEST_TMP_DIR at the prepare test step.

Reply:
Markers aren't doing anything I think, we can just remove them from here.

Reply:
Sorry guys, it's unclear to me, can you tell me exactly what to do with the new files (once I have checked the two steps work), or better push it in my PR?

Reply:
I think you are note using the keyset cache so just ignore me and as umut say you can remove just have
`make test-notebooks` without PYTEST_MARKERS

Reply:
ok, pushing a last time without the markers